cs-engine/1.12/release-notes/prior-release-notes.md

```diff
@@ -11,7 +11,7 @@ title: Release notes archive for Commercially Supported Docker Engine.
 This document contains the previous versions of the commercially supported
```

Comment:
We need to pull out the changes for all the release note files. You can do `git checkout origin/master -- <file-path>` and then `git add <file-path>` to reset those files.